smart_control/reinforcement_learning/scripts/populate_starter_buffer.py

```diff
@@ -46,136 +46,184 @@ def populate_replay_buffer(
-    logger.info(f"Completed all runs, total steps: {total_steps}. Checkpointing buffer one last time...")
     replay_buffer.py_client.checkpoint()
-    logger.info(f"Final replay buffer size: {replay_buffer.num_frames()} frames")
-
-    return replay_buffer
-
-if __name__ == "__main__":
-    import argparse
-    from smart_control.reinforcement_learning.utils.config import CONFIG_PATH
 
-    config_filepath = os.path.join(CONFIG_PATH, 'sim_config_1_day.gin')
-
-    parser = argparse.ArgumentParser(description='Populate a replay buffer with initial exploration data')
-    parser.add_argument('--buffer-name', type=str, required=True, help='Name to identify the saved replay buffer')
-    parser.add_argument('--capacity', type=int, default=50000, help='Replay buffer capacity')
-    parser.add_argument('--steps-per-run', type=int, default=100, help='Number of steps per actor run')
-    parser.add_argument('--num-runs', type=int, default=5, help='Number of actor runs to perform')
-    parser.add_argument('--sequence-length', type=int, default=2, help='Sequence length for the replay buffer')
-    parser.add_argument('--env-gin-config-file-path', type=str, default=config_filepath, help='Environment config file')
-
-    args = parser.parse_args()
-
-    populate_replay_buffer(
-        buffer_name=args.buffer_name,
-        buffer_capacity=args.capacity,
-        steps_per_run=args.steps_per_run,
-        num_runs=args.num_runs,
-        sequence_length=args.sequence_length,
-        env_gin_config_file_path=args.env_gin_config_file_path
-    )
+  # Final checkpoint and stats
+  logger.info(
+      f'Completed all runs, total steps: {total_steps}. Checkpointing buffer'
+      ' one last time...'
+  )
+  replay_buffer.py_client.checkpoint()
+  logger.info(f'Final replay buffer size: {replay_buffer.num_frames()} frames')
+
+  return replay_buffer
+
+
+if __name__ == '__main__':
+  import argparse
+  from smart_control.reinforcement_learning.utils.config import CONFIG_PATH
+
+  config_filepath = os.path.join(CONFIG_PATH, 'sim_config_1_day.gin')
+
+  parser = argparse.ArgumentParser(
+      description='Populate a replay buffer with initial exploration data'
+  )
+  parser.add_argument(
+      '--buffer-name',
+      type=str,
+      required=True,
+      help='Name to identify the saved replay buffer',
+  )
+  parser.add_argument(
+      '--capacity', type=int, default=50000, help='Replay buffer capacity'
+  )
+  parser.add_argument(
+      '--steps-per-run',
+      type=int,
+      default=100,
+      help='Number of steps per actor run',
+  )
+  parser.add_argument(
+      '--num-runs', type=int, default=5, help='Number of actor runs to perform'
+  )
+  parser.add_argument(
+      '--sequence-length',
+      type=int,
+      default=2,
+      help='Sequence length for the replay buffer',
+  )
+  parser.add_argument(
+      '--env-gin-config-file-path',
+      type=str,
+      default=config_filepath,
+      help='Environment config file',
+  )
```

Comment:
let's revert this formatting change and ignore these long lines, to make them more readable